packages/react-examples/src/react-charting/GaugeChart/GaugeChart.Variant.Example.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { DataVizPalette, GaugeChart, GaugeValueFormat } from '@fluentui/react-charting';
+import { DataVizPalette, GaugeChart, GaugeValueFormat, GaugeChartVariant } from '@fluentui/react-charting';
```

Comment:
We should rename this example to single segment.

Reply:
done